os/net/mac/tsch/tsch-conf.h

```diff
@@ -71,14 +71,20 @@
 #define TSCH_DESYNC_THRESHOLD (2 * TSCH_MAX_KEEPALIVE_TIMEOUT)
 #endif
 
-/* Period between two consecutive EBs */
+/* The default period between two consecutive EBs (not taking into account any randomization).
+ * When TSCH_CONF_EB_PERIOD set to 0, sending EBs is disabled completely; the EB process is not started.
```

Comment:
typo: "When TSCH_CONF_EB_PERIOD *is* set"

Reply:
fixed